index.js

```diff
@@ -188,6 +181,14 @@ class Item {
     droppedItem = playerInv.splice(droppedItemIndex, 1).toString();
     return `You dropped the ${this.name}`;
   }
+
+  read() {
+    if (this.name === "note") {
+      return this.description;
+    } else {
+      return "You can't read that";
```

Comment:
//otherwise a message will be returned to the user that the targetted item cannot be read.